### ISSUE TITLE
Crash - while logging in

### ISSUE THREAD

**Issue by xekushuna** [avocode/avocode#305]
Describe your problem here…

-----

```
TypeError: Cannot read property 'headers' of undefined
  at Request._callback (/Users/gabrielizebhigie/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/auth/lib/utils/api.js:1:808)
  at self.callback (/Users/gabrielizebhigie/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/auth/node_modules/request/request.js:354:22)
  at emitOne (events.js:77:13)
  at Request.emit (events.js:166:7)
  at Request.onRequestError (/Users/gabrielizebhigie/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/auth/node_modules/request/request.js:1011:8)
  at emitOne (events.js:77:13)
  at ClientRequest.emit (events.js:166:7)
  at TLSSocket.socketErrorListener (_http_client.js:254:9)
  at emitOne (events.js:77:13)
  at TLSSocket.emit (events.js:166:7)
  at net.js:438:14
  at process._tickCallback (node.js:357:13)

```

**Comment by rychlis** [comment]
This looks like a badly handled network error. Are you by any chance using proxy? Does this happen every time you try to login?

**Comment by rychlis** [comment]
The exception is fixed in Avocode 2.2.1. See our [proxy](https://avocode.zendesk.com/hc/en-us/articles/203402352-How-to-use-a-proxy-server) article if you need that.
